lib/data-types.js

```diff
@@ -345,11 +345,13 @@ inherits(DECIMAL, NUMBER);
 
 DECIMAL.prototype.key = DECIMAL.key = 'DECIMAL';
 DECIMAL.prototype.toSql = function toSql() {
+  let colDefinition = 'DECIMAL';
```

Comment:
I can remove the variable all together, just return `DECIMAL (12, 2)` from first statement other wise  return `DECIMAL`